packages/zosfiles/src/methods/create/Create.ts

```diff
@@ -262,7 +262,7 @@ export class Create {
 
                     case "dsntype": {
                     // Key to create a PDSE.
-                        const type: string = tempOptions.dsntype.toUpperCase();
+                        const type: string = tempOptions.dsntype?.toUpperCase();
```

Comment:
Is the `?` necessary here? I believe this case in the switch statement will only be hit if `dsntype` is defined on the options object.